cli/internal/fs/turbo_json.go

```diff
@@ -107,14 +116,53 @@ func readTurboJSON(path turbopath.AbsolutePath) (*TurboJSON, error) {
 	if err != nil {
 		return nil, err
 	}
+
 	err = jsonc.Unmarshal(data, &turboJSON)
+
 	if err != nil {
 		println("error unmarshalling", err.Error())
```

Comment:
Delete this `println`. If we need extra context with the error, you can use `errors.Wrap` or `errors.Wrapf`